source/_docs/distros.md

```diff
@@ -0,0 +1,94 @@
+  b459145b-8771-4597-8b84-684a3d93dce0   OpenPublic                                    custom   drupal
+  4c7176de-e079-eed1-154d-44d5a9945b65   Empty Upstream                                custom   drupal8
+  d0da49ff-01b3-4351-b53a-a2625b0f8976   Mukurtu CMS                                   custom   drupal
+  86112161-4cb2-410f-8bb1-8a1fb4f56dae   OpenIdeaL                                     custom   drupal
+  216f85b2-620b-470d-9597-f64ade76dc9a   Plato Típico                                  custom   drupal
+  64408272-2d4b-614a-753b-334a2baf4263   Rutgers Unit/Department Website - Wordpress   custom   wordpress
+  8ad1efe0-0231-42ae-9520-c96241495b82   Panopoly                                      custom   drupal
+  8ed60b88-e3bb-4981-9701-777ee9e40435   Test PS Digital                               custom   wordpress
+  50dba1c5-1ff4-4057-a6d5-b31f128894db   Open Ideal 7 (Master)                         custom   drupal
+  31bc4254-be20-4e8d-afe6-6c585e58435a   Atrium                                        custom   drupal
+  53711c64-41eb-4f42-9ca9-a370f8e1e36f   Empty WordPress                               custom   wordpress
+  8a662dde-53d6-4fdb-8eac-eea9f5848d00   Commerce Kickstart                            custom   drupal
+  bf703821-4c18-45a1-88b8-3d9ec302273d   Backdrop                                      custom   backdrop
+  36cf4269-31a6-4f65-a467-f51114715102   Introduction to Theming Basics for Drupal 7   custom   drupal
+  3b754bc2-48f8-4388-b5b5-2631098d03de   CiviCRM Starter Kit                           custom   drupal
+  bc77fa2f-2235-4eec-8e6b-4d69d1cf5908   OpenPublish                                   custom   drupal
+  de858279-cb87-4664-825c-fcb4c2928717   Static HTML                                   custom   unknown
+  d7370d7e-46fb-4b10-b79f-942b5abf51de   DKAN                                          custom   drupal
+  4fa4f7d1-e341-48f7-9594-4e7c21b9bb68   Drupal8 PHP                                   custom   drupal8
+  5b70ea4b-8ba3-4c78-9cf2-8086a5524885   Open Academy                                  custom   drupal
+  bd5bac3b-2ed4-4b0b-b085-b9cfb574940f   Open Enterprise (Deprecated)                  custom   drupal
+  10d6937e-1dd2-4490-9950-11867ba43597   RedHen Raiser                                 custom   drupal
+  21e1fada-199c-492b-97bd-0b36b53a9da0   Drupal 7                                      custom   drupal
+  2adff196-4672-44c9-af2a-4590963b90d8   OpenAid                                       custom   drupal
+  f575e77a-ff49-4bab-a53c-dbf8f59247bc   Empty Drupal 7                                custom   drupal
+  0fce9338-bd6f-4b87-a489-e9928a027696   Elephant Ventures Drupal Patterns             custom   drupal
+  974b75c2-4ba7-49f8-8a54-3a45c07dfe02   Drupal 6                                      custom   drupal
+  c28628b2-6b8f-4962-9838-69295c3fff68   Fairfax County Public Schools                 custom   drupal8
```

Comment:
Can we verify this list is accurate?  For example, I'm not sure if Fairfax County Public Schools should be there.  It's one agency's custom upstream.